src/Couchbase.Lite.Shared/Serialization/FLValueConverter.cs

```diff
@@ -190,7 +190,9 @@ private static object ToObject(FLValue* value, Database db, int level = 0, Type
                         retVal[key] = ToObject(Native.FLDictIterator_GetValue(&i), db, level + 1, hintType1);
                     } while (Native.FLDictIterator_Next(&i));
 
-                    return ConvertDictionary(retVal as IDictionary<string, object>, db) ?? retVal;
+                    return (string)retVal["@type"] == "blob" 
```

Comment:
We have constants set up for this kind of stuff ("@type" and "blob") but they are not needed here.  `ConvertDictionary` already checks this.